CONTRIBUTING.rst

```diff
@@ -277,3 +286,22 @@ Get your fix accepted into the master branch
    with it, it'll be accepted and merged onto the master branch.
 
 .. _Zuul Dashboard: http://3ci-zuul.ciscolabs.net
+
+Only run a subset of tests in the experimental queue
+----------------------------------------------------
+
+While developing new jobs or modifying existing ones it is recommended to only
+run the tests you are creating/modifying in the experimental queue in order to
+speed up test run times and reduce load on the CI.
+
+To do this you can add a regex pattern to the commit message of your [DNM]
+patch on gerrit to tell the third party CI to only run certain jobs. In your
+commit message add the line ``Cisco-CI-Experimental-Regex: <regex-pattern>``
+(where <regex-pattern> matches the jobs you want to run). An example can be
+seen - https://review.openstack.org/#/c/542324/
+
+Common examples:
+
+- Only run ASR1K - ``Cisco-CI-Experimental-Regex: .*-asr1k-.*``
+- Only run UCSM - ``Cisco-CI-Experimental-Regex: .*-ucsm-.*``
+- Only run Nexus - ``Cisco-CI-Experimental-Regex: .*-nexus-.*``
```

Comment:
With the Regex applied the "skipped" jobs will show up as "failed" in the results, and it is expected that at least one final run with all the jobs enabled is run before we merge the changes to project-config-third-party to ensure there are no regressions in the other jobs caused by the changes.